ionoscloud/resource_autoscaling_group_test.go

```diff
@@ -534,13 +538,14 @@ resource ` + constant.AutoscalingGroupResource + `  ` + constant.AutoscalingGrou
       image_alias = "ubuntu:latest"
       name        = "volume_1"
       size        = 10
-      ssh_keys    = ["` + sshKey + `"]
       type        = "HDD"
       boot_order  = "AUTO"
+      ssh_keys       = ["` + sshKey + `"]
+      image_password = random_password.image_password.result
```

Comment:
Initially, the test was supposed to check removal of optional fields
`image_password` is also an optional field, but it seems removing this field caused a `plan was not empty` failure.
Currently we are unsure why, but it seems to be related to the `Sensitive` attribute, as removing this also solves the issue.